### DISCUSSION TITLE
Customize Hangfire Dispatcher Channels

### DISCUSSION THREAD

**Discussion by mthrift2100** [elsa-workflows/elsa-core#3932] [Q&A]
Can the WorkflowChannelOptions class be used to customize the Hangfire Queue names that are used to dispatch activities?  We plan to run Elsa server on multiple apps, each app is also using the Hangfire Server.  But all apps will point to the same persistence database (for both Hangfire and Elsa).

But each app will only be de-queueing work from its own queue.  When using Elsa to dispatch activities through Hangfire however, the queue name is generic.  But I see where the queue name is appended with the Workflow Channel.  

Can this be customized so that each app that uses Elsa Server can have a different set of queues?

**Comment by sfmskywalker** [comment]
I believe that right now, it's only used to create variations on queues created with Rebus, but the Hangfire module can probably be updated to take advantage of the channel options as well. Perhaps you want to take a crack at it and provide a PR?
